lib/src/codecs/codec_IMUEvent.cpp

```diff
@@ -86,4 +86,38 @@ std::string IMUevent::getType() const
     return IMUevent::tag;
 }
 
+double IMUevent::convertToSI(int index, int value)
+{
+    static constexpr double ay_bias = 0.12;
```

Comment:
Bias and gains coming from individual sensor calibration - would be better to bring those in via a config file.

Reply:
agreed. that won't happen in this PR though and should be done in a second update when required